src/swap/useFilterChips.ts

```diff
@@ -23,20 +22,6 @@ export default function useFilterChip(selectingField: Field | null): FilterChip<
   ).popularTokenIds
   const supportedNetworkIds = getSupportedNetworkIdsForSwap()
 
-  const networkIdFilters =
-    supportedNetworkIds.length > 1
```

Comment:
should we retain this condition? seems very unlikely we go back to a single network though

Reply:
Yea, i thought the additional code for a *slightly* better experience wasn't worth it since, like you said, seems very unlikely we will return to single network.